app/views/transferConfirmation.scala.html

```diff
@@ -0,0 +1,53 @@
+@()(implicit messages: Messages)
```

Comment:
How do you get to this page? Was there already a controller and route pointing to it?

Reply:
When testing this page, I was just changing the dashboard controller from `Ok(views.html.dashboard())` to  `Ok(views.html.transferConfirmation())`. Since the Jira task was to just "Placeholder page for transfer confirmation page", I wasn't sure if I should add more functionality to it since there is another card that is "Add Transfer Summary page in Play app". Should I also add a route for it and create a controller for it?
According to the wireframes, this page is the 8th page, but we've only built 5 (I think Loki is working on the 6th and the 7th is the metadata page), so there wouldn't be a direct page to it for now, you'd have to type the url in (if I build the controller and add a route for it).

Reply:
Yeah, if we're adding a page then we should have a route and controller to navigate to it so that it's easy to link to from the other task, and so that we can add more features to it. 🙂